source/core/Variants.cpp

```diff
@@ -37,6 +37,7 @@ VariantType::VariantType(TypeManagerRef typeManager)
     _topAQSize = TypeManager::HostPointerToAQSize();
     _aqAlignment = TypeManager::AQAlignment(sizeof(void*));
     _encoding = kEncoding_Variant;
+    _isValid = 1;
```

Comment:
Lack of this field was causing a waveform test to fail. The readJSON API verifies that the typeref being read is "valid" before reading it.